docs/self-managed/setup/deploy/azure/microsoft-aks/aks-helm.md

```diff
@@ -0,0 +1,328 @@
+```
+
+### Export database values
+
+When using standard authentication (username and password), specific environment variables must be set with valid values. Follow the guide for [Terraform](./terraform-setup.md#set-up-azure-authentication) to set them correctly.
+
+Verify the configuration of your environment variables by running the following loop:
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/main/azure/kubernetes/aks-single-region/procedure/check-env-variables.sh
+```
+
+## Deploy Camunda 8 via Helm charts
+
+For more configuration options, refer to the [Helm chart documentation](https://artifacthub.io/packages/helm/camunda/camunda-platform#parameters). Additionally, explore our existing resources in the [Camunda 8 Helm chart](/self-managed/setup/install.md) and [related guides](/self-managed/setup/guides/guides.md).
+
+Depending on your installation path, you may use different settings.
+For easy and reproducible installations, we will use YAML files to configure the chart.
+
+### 1. Create the `values.yml` file
+
+Start by creating a `values.yml` file to store the configuration for your environment. This file will contain key-value pairs that will be substituted using `envsubst`. You can find a reference example of this file here:
+
+```yaml reference
+https://github.com/camunda/camunda-deployment-references/blob/main/azure/kubernetes/aks-single-region/helm-values/values-no-domain.yml
+```
+
+#### Reference the credentials in secrets
+
+Before installing the Helm chart, create Kubernetes secrets to store the Keycloak database authentication credentials.
+
+To create the secrets, run the following command:
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/main/azure/kubernetes/aks-single-region/procedure/create-external-db-secrets.sh
+```
+
+### 2. Configure your deployment
+
+#### Enable Enterprise components
+
+Some components are not enabled by default in this deployment. For more information on how to configure and enable these components, refer to [configuring Web Modeler, Console, and Connectors](../../../install.md#configuring-web-modeler-console-and-connectors).
+
+#### Elasticsearch options
+
+Camunda Helm chart supports both internal and external Elasticsearch deployments. For production workloads, we recommend using an externally managed Elasticsearch service (for example, [Elastic Cloud on Azure](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/elastic.ec-azure-pp)). Terraform support for Elastic Cloud on Azure can be restrictive but remains a viable option. In this guide, we default to the internal deployment of Elasticsearch.
+
+<details>
+<summary>Show configuration to enable internal Elasticsearch</summary>
+
+```yaml
+global:
+  elasticsearch:
+    enabled: true
+  opensearch:
+    enabled: false
+
+elasticsearch:
+  enabled: true
+```
+
+</details>
+
+#### Use internal PostgreSQL instead of the managed PostgreSQL service
```

Comment:
This is a bit odd. Can we clarify why this is needed. What service depends on it? User without much background info will get confused.